src/sentry/analytics/events/first_event_sent.py

```diff
@@ -2,18 +2,15 @@
 
 
 # first error for an organization
+@analytics.eventclass("first_event.sent")
 class FirstEventSentEvent(analytics.Event):
-    type = "first_event.sent"
-
-    attributes = (
-        analytics.Attribute("user_id"),
-        analytics.Attribute("organization_id"),
-        analytics.Attribute("project_id"),
-        analytics.Attribute("platform", required=False),
-        analytics.Attribute("url", required=False),
-        analytics.Attribute("has_minified_stack_trace", required=False),
-        analytics.Attribute("project_platform", required=False),
-    )
+    user_id: int
+    organization_id: int
+    project_id: int
+    platform: str | None = None
+    url: str | None = None
+    has_minified_stack_trace: str | None = None
```

Comment:
I think this should be a bool, and we'll want to change the caller for FirstEventSentEventForProject as well?